components/support/sync15-traits/src/bridged_engine.rs

```diff
@@ -2,6 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+use std::{error::Error, fmt};
+
```

Comment:
lots of blank lines :)

Reply:
Haha, yeah! I've been grouping imports into sections (`std` first, then other crates, then our `crate::` imports), but it's kind of unnecessary when there's one line each! 😅